modules/openapi-generator/src/main/resources/lumen/composer.mustache

```diff
@@ -4,7 +4,7 @@
     "version": "{{artifactVersion}}",
     {{/artifactVersion}}
     "description": "{{description}}",
-    "keywords": [
+    "keywords": ["framework", "laravel", "lumen",
```

Comment:
I think it is better to align the keywords vertical like below:

```json
    "keywords": [
         "framework",
         "laravel",
         "lumen",
         "openapi",
```

Reply:
updated. thanks!